qdrant-landing/themes/qdrant/static/css/blocks/_documentation.scss

```diff
@@ -29,15 +29,15 @@
     }
 
     h4 {
-      font-size: 1.25rem;
+      font-size: 1.1rem;
     }
 
     h5 {
-      font-size: 1.25rem;
+      font-size: 1rem;
     }
 
     h6 {
-      font-size: 1.25rem;
+      font-size: 1rem;
     }
```

Comment:
I've updated the H4/H5/H6 sizes in our documentation. Otherwise the subsection I've added has the same font size as its parent which is quite confusing.

The alternative was to use just regular **bold** text as a header. But I like using an H4 with the above change better.